### PR TITLE
Mongo: Fix reversing polygon coordinates

### PR DESCRIPTION
**Pull request by dplewis**
Closes: https://github.com/parse-community/parse-server/issues/4607

Has been tested on the latest parse-dashboard.

https://docs.mongodb.com/manual/reference/glossary/#term-wgs84

### CODE REVIEW

**Comment by dplewis** [comment]
@JacobJT What you see in the dashboard and input into your objects are in lat/lng. The databases are saved as lng/lat.(Postgres already stored at lng/lat)

I didn't realize until today that I haven't updated my dashboard to support type Polygon which is why I missed this issue. 

@flovilmart How does this look?

File: spec/MongoTransform.spec.js [hunk 1]
```diff
@@ -157,13 +157,16 @@ describe('parseObjectToMongoObjectForCreate', () => {
   });
 
   it('polygon', (done) => {
-    const input = {location: { type: 'Polygon', coordinates: [[[45, -45],[45, -45]]]}};
+    const lat = -45;
+    const lng = 45;
+    // Mongo stores polygon in WGS84 lng/lat
+    const input = {location: { type: 'Polygon', coordinates: [[[lng, lat],[lng, lat]]]}};
```
Comment (flovilmart) [inline spec/MongoTransform.spec.js]: this is against the documentation for the GeoPoints,m where the list of corrdinates should be provided as lat,lng pairs, not lng,lat pairs.

http://parseplatform.org/Parse-SDK-JS/api/v1.11.1/Parse.GeoPoint.html

If the storage need to be reveresed, that's another issue but should not for something un natural for the user.

Comment (dplewis) [reply spec/MongoTransform.spec.js]: I wrote it this way to highlight that the input and output are going to be reversed.

The current GeoPoint transform tests are slightly confusing at a glance.

Shouldn't this be testing `mongoObjectToParseObject` input -> lng/lat output -> lat/lng?

Comment (flovilmart) [reply spec/MongoTransform.spec.js]: OK I misread. Do you have a test that ensures the input / output orders are preserved? ie the mongo transformation is properly applied and reversed?

**Comment by dplewis** [comment]
@JacobJT Thanks for helping with this one.

**Pull request merged**